NetWork.md

```diff
@@ -0,0 +1,142 @@
+# The (dwyl) NetWork [_Work-in-Progress_]
+
+A network of creative technology people engaged
+in learning, fulfilling/innovative work
+with a socially beneficial outcome.
+
+## _Why_?
+
+There is no shortage of "professional networks",
+LinkedIn being the most prominent one.
+On LinkedIn people _pay_ for "Premium"
+which is $10/month and allows ...
+> Please summarize the list of Premium "features". `help wanted`
+
+LinkedIn could do a _whole_ lot more with the community,
+but because they have steady revenues they have all but stopped innovating
+(_we don't count occasional UI changes as "innovation"..._).
+
+> _See below for "**The Problem with LinkedIn**"_.
+
+## _What_?
+
+The Creative Technology Rapid Learning Community.
+Connect and learn with people who are experienced/knowledgeable
+in the field you are interested in.
+
+### Content Will _Always_ Be Free
+
+All _recorded_ content is (_and always will be_) free.
+This is essential for ensuring ***accessibility***
+to people who cannot (_yet_) afford to pay for learning.
+
+>_The **last** thing we want is to create a "pay wall"
+like FrontEnd Masters or TreeHouse
+  which are **way** out of reach for unemployed
+  people in low-income countries!_
+
+Once people feel a bit more confident with their knowledge/skills,
+and are getting _income_ from their chosen skill they will
+naturally want to "_support the network_" with a _nominal_
+payment. The payment will grant them _considerable_ additional value/benefit.
+
+### Paid Value-Added Services (_Which Cost Money to Provide_)
+
++ Connect with fellow creative technology professionals
+without fear of being spammed by recruiters.
++ Live-streams learning workshops and meetups.
++ Access to ask questions of the highly experienced/skilled people
+delivering the workshop in real-time.
++ Request personal advice from industry insiders.
++ Get notified of potential work by NetWork Members.
++ Meet and interact with other creative people online and in-person.
+
+
+
+## _Who_?
+
+People who want to learn and do
+creative technology work professionally.
+
+
+# _How_?
+
+## Membership
+
+How does someone _become_ a member?
+People can either _apply_ in which case they need to submit
+
+
+## Membership Fees?
+
+If we charge $10/month it might sound "_reasonable_"
+to most people in G20 ("_developed_") countries where average earnings are
+comparatively _high_ by international standards.
+But if your _income_ is $100/month (_e.g: in Manila or Mobassa)
+then $10 is prohibitively expensive
+(_even if you get a 10x the value of the membership fee it's still a lot
+in the beginning for someone on a lower income..!_)
+
+### 1% of Minimum Wage?
+
+> _**1% of National Minimum Wage** for the country you are based in_.
+
+If **Minimum Wage** is **€650**/month (_e.g. Portugal_)
```

Comment:
lol. hardly. it's just the only way I could think of to make the `NetWork` affordable to everyone in the world using "Purchasing Power Parity". 😉